src/coreComponents/physicsSolvers/contact/ContactFields.hpp

```diff
@@ -108,15 +108,15 @@ DECLARE_FIELD( dTraction_dPressure,
 DECLARE_FIELD( fractureState,
                "fractureState",
                array1d< integer >,
-               FractureState::Stick,
+               FractureState::Open,
```

Comment:
Why do you need to change the default initial status?

Reply:
This is actually something I wanted to discuss with you.

For the following problem (see image), the `LagrangianContact` solver (no poromechanics), starting with a stick condition as an initial guess the solution at the first iteration has some `NaN` which makes me think that the system is singular. Any idea why that may be the case? Maybe, one thing to do, is to not make everything crash if a `NaN` shows up but rather try a different configuration but I would like to understand if it's expected first.

![simpleSetup](https://user-images.githubusercontent.com/49037133/205709594-e8ce1168-3a25-49f1-bd2e-e8a03e2871a4.png)

Reply:
In my experience, `NaN` values occur when a part of the body becomes unconstrained during the simulation. For example, if the right side was constrained only by the fracture and this opens, then it is free to move (at least in some directions) and `NaN` appears. However, when the contact surface is in stick mode there is no way to justify the `NaN` other than as an error.

Would you mind sharing the input files used to run the problematic simulation?

Reply:
@andrea-franceschini, have you had a chance to test this?

Reply:
Yes! Sorry @CusiniM for the delay.

The problem is the test case: it is too small. All nodes on the fracture are constrained and there is only one fracture element. This means that:
1) the fracture cannot _communicate_ with the surrounding 3D body
2) the stabilization cannot be built (it is a jump penalization, but since there is only one fracture element, there are no jumps)

If you refine the mesh a bit, increasing the number of elements from 2x1x1 to 4x2x2, the test case runs fine.

This is the new input, working with `fractureState` initialized as `Stick`. (It is a txt just because github complains about xml).
[conformingFracture_2d_contactOnly.txt](https://github.com/GEOSX/GEOSX/files/10225414/conformingFracture_2d_contactOnly.txt)

Reply:
Hmm, yeah I had noticed that but I was trying to set up the simplest possible case. 
- Maybe the issue comes from the fact that the fracture nodes are not allowed to move in the vertical direction so the "slip" is completely blocked?
- I don't see why the stabilization would matter. For a single element it should not be necessary IMO.

Reply:
You are right! The lack of stabilization is not the problem.
The key point is to choose a different pattern for the constraints. The fracture has to be connected to the 3D body.
If only the two faces normal to the x-direction are fully constrained, the example runs fine.

[conformingFracture_2d_contactOnly_constrained.txt](https://github.com/GEOSX/GEOSX/files/10235993/conformingFracture_2d_contactOnly_constrained.txt)

Reply:
If that's the case I think this PR can be wrapped up and merged and then we can start some testing.

Reply:
I agree.
Thanks @CusiniM for tacking care of this PR!